packages/react-magma-dom/src/i18n/default.ts

```diff
@@ -107,7 +107,7 @@ export const defaultI18n: I18nInterface = {
     startOfWeek: 'sunday',
     calendarIconAriaLabel: 'Toggle Calendar Widget',
     calendarOpenAnnounce:
-      'Calendar Widget is now open. Press the question mark key to get the keyboard shortcuts for changing dates.',
+      'Calendar Widget is now open. Press the question mark key to get the keyboard shortcuts for changing dates. If you are using the NVDA screen reader, you may need to switch it to Focus Mode for proper interaction with the calendar.',
```

Comment:
We should remove `Press the question mark key to get the keyboard shortcuts for changing dates.` since we removed that shortcut in a previous ticket but missed this line.